qa-covid19.planx-pla.net/portal/gitops.json

```diff
@@ -1264,28 +1264,16 @@
         ],
         [
           {
-            "title": "Cook County Daily Cases",
-            "description": "Here we plot daily case counts over time, where the red bars are reported cases and the blue curve traces the estimated number of true (i.e., reported and not-reported) cases. The darker and lighter shaded regions correspond to 50% and 95% posterior credible intervals, respectively.",
+            "title": "Cook County Daily New Cases and Forecast",
+            "description": "Here we plot daily case counts over time in Cook county, where the red line is reported cases and the grey shaded area depicts the inferred posterior predicted cases. Forecasts (blue shaded area) show how new case numbers would have developed (95% Bayesian credible intervals).",
```

Comment:
you mention a red line but i don't see it

<img width="561" alt="Screen Shot 2021-11-09 at 11 35 42 AM" src="https://user-images.githubusercontent.com/4224001/140975662-27905677-b0ed-4935-b21a-71a11d235578.png">

Reply:
nvm, this is the test chart 🙂 ✅